src/backend/dx11/src/device.rs

```diff
@@ -1246,16 +1246,18 @@ impl hal::Device<Backend> for Device {
             unsafe { ComPtr::from_raw(swapchain) }
         };
 
-        let images = (0..config.image_count).map(|i| {
+        // TODO: for now we clamp to 1 buffer..
```

Comment:
why only one?

Reply:
Right now the swapchain is hardcoded to only use the `DXGI_SWAP_EFFECT_DISCARD` which as I understand it can only handle 1 backbuffer resource. All the others require w7+ which can be added in later: https://msdn.microsoft.com/en-us/library/windows/desktop/bb173077(v=vs.85).aspx